stylesheets/_component.timeline.scss

```diff
@@ -0,0 +1,585 @@
+        height: 40px;
+        top: 0;
+        width: 40px;
+    }
+
+    span {
+        display: block;
+        font-size: 10px;
+        line-height: 2.1;
+        text-align: center;
+
+        @include respond-min($screen-phone) {
+            font-size: 13px;
+        }
+
+        @include respond-min($screen-tablet) {
+            font-size: 20px;
+        }
+    }
+
+    &__image {
+        border-radius: 50%;
+        height: 20px;
+        width: 20px;
+
+        @include respond-min($screen-phone) {
+            height: 25px;
+            width: 25px;
+        }
+
+        @include respond-min($screen-tablet) {
+            height: 40px;
+            width: 40px;
+        }
+    }
+
+    &--user {
+        background-color: #75ce66;
+        color: color(white);
+    }
+
+    &--admin {
+        background-color: color(jadu-blue);
+        color: color(white);
+    }
+
+    &--system {
+        background-color: color(jadu-blue);
+        color: color(white);
+    }
+}
+
+.timeline__message-box {
+    .timeline-content {
+        background-color: darken(#fafafa, 5%);
+
+        @include respond-min($screen-tablet) {
+            &::after {
+                border-color: rgba(136, 183, 213, 0);
+                border-right-color: darken(#fafafa, 5%);
+                border-width: 7px;
+                margin-top: -7px;
+            }
+        }
+    }
+
+    .timeline-avatar {
+        display: none;
+
+        @include respond-min($screen-tablet) {
+            display: block;
+        }
+    }
+}
+
+.timeline-content {
+    background: #fafafa;
+    border: 1px solid $timeline-content-border;
+    padding: 1em;
+
+    @include respond-min($screen-tablet) {
+        border: 1px solid $timeline-content-border;
+        margin-left: 70px;
+        margin-top: $padding-base;
+        position: relative;
+
+        &::before,
+        &::after {
+            border: solid transparent;
+            content: '';
+            height: 0;
+            position: absolute;
+            right: 100%;
+            top: 16px;
+            width: 0;
+        }
+
+        &::after {
+            border-color: rgba(136, 183, 213, 0);
+            border-right-color: #fafafa;
+            border-width: 7px;
+            margin-top: -7px;
+        }
+
+        &::before {
+            border-color: rgba(194, 225, 245, 0);
+            border-right-color: $timeline-content-border;
+            border-width: 8px;
+            margin-top: -8px;
+        }
+    }
+
+    &__title {
+        color: #303e49;
+        display: inline-block;
+        margin: 0 0 0 26px;
+
+        @include respond-min($screen-phone) {
+            margin: 0 0 0 32px;
+        }
+
+        @include respond-min($screen-tablet) {
+            line-height: $line-height-base;
+            margin: 0;
+            width: 60%;
+        }
+    }
+
+    &__note-title {
+        margin: 1em 0;
+    }
+
+    &__text {
+        @include word-wrap();
+
+        margin: .5em 0;
```

Comment:
selector specific styles should be above includes

Reply:
Scssllint complaints with: "Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets". Therefore should remain as is